test/papi.json

```diff
@@ -0,0 +1,10 @@
+{
+  "relaychain": {
+    "wsUrl": "ws://127.0.0.1:39459",
+    "metadata": "relaychain.scale"
+  },
+  "storagehub": {
+    "wsUrl": "ws://127.0.0.1:42933",
+    "metadata": "storagehub.scale"
```

Comment:
Do these scale files get autogenerated somehow? Or do we have to run something everytime there is an update in the SH runtime?

Reply:
Nevermind, found it in the docs. Good work on that.